functions/meme-getter/tests/memeGetter.test.js

```diff
@@ -0,0 +1,38 @@
+import * as MemeService from '../../../services/meme-maker';
+import { handler } from '../';
+import data from '../data.json';
+import * as utils from '@utils';
+import { mockData } from '@utils/mockData';
+
+const {
+	MOCK_JOKE_SUCCESS_RESPONSE: mockSuccessResponse,
+	MOCK_JOKE_FAILURE_RESPONSE: mockFailureResponse,
+} = mockData;
+
+describe('testing memeGetter handler', () => {
+	let event;
+	let result;
+	beforeAll(() => {
+		event = data;
+	});
+	it('should callled the handler', async () => {
+		const memeSpy = jest
+			.spyOn(MemeService, 'getMemes')
+			.mockResolvedValue(mockSuccessResponse);
```

Comment:
I think it's always nicer to use `mockResolvedValueOnce` test blocks should be isolated pieces of code and it just fits the bill

Reply:
Done